tokio-trace/tokio-trace-core/src/field.rs

```diff
@@ -216,3 +495,87 @@ impl Iterator for Iter {
         })
     }
 }
+
+// ===== impl ValueSet =====
+
+impl<'a> ValueSet<'a> {
```

Comment:
There are several more functions that I think this type _could_ provide, but I didn't implement them for now because I wanted to stick with the minimum necessary public API. 

For example, we could easily have:
- an iterator,
- `get(&Field) -> Option<&Value>` --- this might potentially be useful if a subscriber knows up front that there's only one field that it wants to record, however, I don't know if it's strictly necessary.
- pub access to the `FieldSet`/all its methods